extensions/localPublish/src/index.ts

```diff
@@ -31,6 +31,7 @@ interface RunningBot {
   status: number;
   result: {
     message: string;
+    runtimeLogs?: string;
```

Comment:
the name suggests it should be an array, is it?

Reply:
its not an array. It's just stdout from the runtime that comes in batches. Makes sense to make it singular.